bad.js

```diff
@@ -0,0 +1,4 @@
+var a = function(){
```

Comment:
One (or more) spaces required before opening brace for block expressions
Missing space before opening curly brace